app/src/main/java/ar/com/wolox/android/training/model/User.java

```diff
@@ -0,0 +1,10 @@
+package ar.com.wolox.android.training.model;
+
+/**
+ * User model, contains credentials from logged person
+ */
+public class User {
+
+    public String user;
```

Comment:
Since Java works with OOP, we should take care of encapsulation. The attributes should be private and there should exists getters and setters if necessary.

Also, create a constructor to initialize it with all the attributes already setted.